### ISSUE TITLE
Dashboard - Orders with no customers

### ISSUE THREAD

**Issue by inssein** [Sylius/Sylius#4601]
The dashboard order display recently broke, and I was going to quickly fix it, but from what I see the, the current admin UI is going away.

The fix seems pretty simple (modify DashoardController and join orders with customer) so that the display doesn't error out. Should I submit it as a PR?

Also, I see the code committed for the new AdminBundle, but I don't see the routes registered anywhere. How are you guys working on it?

**Comment by pjedrzejewski** [comment]
https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/AdminBundle/Resources/config/routing/tax_category.yml

http://docs.sylius.org/en/latest/bundles/SyliusResourceBundle/routing.html

:)

**Comment by inssein** [comment]
@pjedrzejewski ah, I realized it was because I don't have some of the more recent routing changes since they are not in Sylius-Standard yet. I went ahead and made the changes. 

I now need admin assets, and I see its a matter of copying the package.json into my project, installing, and running the gulp commands. However. this seems cumbersome, since if the gulp file changes, I will be manually copying in changes. What is a good way of going about this?
